### PR TITLE
Looking up (closes #615) and climbing (closes #621)

### LINKED ISSUE #615: Реализовать возможность "смотреть наверх" через тень
Для начала просто показывать игроку глазами "тени" то, что происходит на вернем уровне. Позже можно ограничивать "глубину" того, как далеко видно от опенспейсов.

### LINKED ISSUE #621: Добавить возможность карабкаться на верхние уровни
Если выполняется одно из следующих условий:
1) Персонаж стоит на столе, стуле, кровати, шкафу или ряде других крупных обьектов.
2) Персонаж держит в руках инженерное оборудование для этого, например какую кошку.
3) Гравитации нет.

То посмотрев наверх (#615), персонаж может кликнуть на тайл или кетволк, находящийся рядом с его тенью, и если на тайле ничего не блокирует движение, то взобраться на него.

### PR DESCRIPTION
**Pull request by Joodka**
Closes <a href="https://github.com/Endless-Horizon/CEV-Eris/issues/615">Добавить возможность карабкаться на верхние уровни #615</a>
And <a href="https://github.com/Endless-Horizon/CEV-Eris/issues/621">Реализовать возможность "смотреть наверх" через тень #621</a>

Players can observe levels above by shadow's eyes. And climb onto floors by clicking them (with some conditions).

With gravity turned off:

With some types of objects under mob:

### CODE REVIEW

File: code/_onclick/other_mobs.dm [hunk 1]
```diff
@@ -29,6 +29,36 @@
...
+				/obj/structure/closet/crate
+			)
+			var/atom/helper = null
+			var/area/location = get_area(loc)
+			if(!location.has_gravity)
+				helper = src
+			else
+				for(var/type in objects_to_stand_on)
+					helper = locate(type) in src.loc
+					if(helper) break
```
Comment (fluorescent-net) [inline code/_onclick/other_mobs.dm]: Please use multiline condition blocks.

File: code/_onclick/other_mobs.dm [hunk 2]
```diff
@@ -29,6 +29,36 @@
...
+				if(!helper) return
```
Comment (fluorescent-net) [inline code/_onclick/other_mobs.dm]: Please use multiline condition blocks.

File: code/modules/mob/living/carbon/human/human.dm [hunk 3]
```diff
@@ -1512,3 +1512,23 @@ var/list/rank_prefix = list(\
...
+		var/turf/above = GetAbove(src)
+		if(istype(above, /turf/simulated/open))
+			src << "<span class='notice'>You look up.</span>"
+			if(client)
+				reset_view(shadow)
+			return
+		src << "<span class='notice'>You can see [above].</span>"
+	else
+		src << "<span class='notice'>You can't do it right now.</span>"
+	return
```
Comment (fluorescent-net) [inline code/modules/mob/living/carbon/human/human.dm]: Please add empty line to end of file.

**Comment by fluorescent-net** [comment]
Could you please check all other actions in RangedAttack proc after your code (laser eyes/tk)?

**Comment by Joodka** [comment]
Works properly.
And now we can climb onto catwalks the same way.

**Comment by Vivalas** [comment]
Holy shit this is :ok_hand: Russians FTW.

**Pull request merged**